provider/pkg/await/deployment_test.go

```diff
@@ -664,12 +665,12 @@ func Test_Apps_Deployment_Without_PersistentVolumeClaims(t *testing.T) {
 				// The Deployment specifically does not reference the PVC in
 				// its spec. Therefore, the Deployment should succeed no
 				// matter what phase the PVC is in as it does not have to wait on it.
+				deployments <- watchAddedEvent(
```

Comment:
The event is reordered since we're using unbuffered channels which will block until there is a value receiver.